mfr/server/handlers/export.py

```diff
@@ -58,6 +58,13 @@ class ExportHandler(core.BaseHandler):
     async def get(self):
         """Export a file to the format specified via the associated extension library"""
 
+        # File is already in the requested format
+        if self.metadata.ext == ".{}".format(self.format):
+            await self.write_stream(self.provider.download())
```

Comment:
FYI, my first thought is to add a `try` here but then I realize the failure should be thrown if download fails.